src/Html/AssetHtmlGenerator.php

```diff
@@ -144,6 +152,11 @@ public function linkAssets (array $assetPaths, bool $withDependencies = true) :
                 }
             }
 
+            if ($this->allowCors)
+            {
+                $embed->setAttribute("crossorigin", "crossorigin");
```

Comment:
That is not a valid value for this attribute, see [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin).

You should use:

```suggestion
                $embed->setAttribute("crossorigin", "anonymous");
```